vector/src/main/java/im/vector/fragments/VectorSettingsPreferencesFragment.kt

```diff
@@ -801,6 +805,23 @@ class VectorSettingsPreferencesFragment : PreferenceFragmentCompat(), SharedPref
             false
         }
 
+        mUseDefaultStunPreference.let {
+            activity?.let { activity ->
+                it.isChecked = PreferencesManager.useDefaultTurnServer(activity)
```

Comment:
This line is not necessary

Reply:
It's to be sure to take the correct default value (otherwise it's declared in the pref.xml and in the PrefManager)